README.md

```diff
@@ -1,13 +1,12 @@
-# gatekeeper
+# Gatekeeper
 
 [![Build Status](https://travis-ci.org/open-policy-agent/gatekeeper.svg?branch=master)](https://travis-ci.org/open-policy-agent/gatekeeper) [![Docker Repository on Quay](https://quay.io/repository/open-policy-agent/gatekeeper/status "Docker Repository on Quay")](https://quay.io/repository/open-policy-agent/gatekeeper)
 
 ## Warning: Restructing underway
 
-This is a new project that is undergoing heavy restructuring.  The policy format, architecture, interfaces, and code layout are all subject to change.
+This is a new project that is under development.  The architecture, interfaces, and code layout are all subject to change, though the policy syntax and ConstraintTemplate spec schema should be stable enough for alpha.
```

Comment:
I think a short description of what `Constraints` and `ConstraintTemplates` are would be very helpful here.

Reply:
Don't want to get too in the weeds on the "warning" section, but adding a pointer to the appropriate section of the doc if they want more info on what these things are.